src/accordion/index.jsx

```diff
@@ -5,7 +5,9 @@ import castArray from 'lodash/castArray';
 class Accordion extends React.Component {
 
   componentDidMount() {
-    const open = this.props.children.map(() => false);
+    const open = this.props.children.map(child => {
+      return (child.props && child.props.isOpen) ? child.props.isOpen : false;
+    });
```

Comment:
This was preventing ExpandingPanel inside the Accordion component from being set as open on page load.

If there is a validation fail on the condition form submit then we want to be able to keep the condition expanded so you can see the error message.